### DISCUSSION TITLE
How to train on my own dataset

### DISCUSSION THREAD

**Discussion by Caper6666** [RapidAI/RapidLayout#2] [Q&A]
非常感谢你的工作，我希望可以在我自己的数据集上进行微调，我该怎么做呢，只用替换对应的模型文件么，比如paddle的onnx表格识别模型，我在paddle的网站上没有找到如何转换为onnx格式的方式，期待你的回复。

**Comment by SWHL** [comment]
这就是一个目标检测任务，直接用[yolov8](https://github.com/ultralytics/ultralytics)训练，用官方提供接口转换为onnx,即可接入本项目
